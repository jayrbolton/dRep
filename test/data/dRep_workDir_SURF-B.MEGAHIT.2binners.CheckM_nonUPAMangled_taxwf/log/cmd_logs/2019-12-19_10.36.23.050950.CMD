/usr/local/bin/mash paste /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/MASH_files/ALL.msh /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/MASH_files/sketches/chunk_0/chunk_all.msh
