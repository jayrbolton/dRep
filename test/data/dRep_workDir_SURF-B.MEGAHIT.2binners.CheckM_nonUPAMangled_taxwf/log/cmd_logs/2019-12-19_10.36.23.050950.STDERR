Writing /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/MASH_files/ALL.msh...
